Cookbook/Technical-Documents/OutOffOfficeRequest.md

```diff
@@ -0,0 +1,60 @@
+# Requesting and Notifying Out Of Office time
+
+Someone who is Out Of Office means that is not in their usual work place at their usual working hours. This can be when someone goes on holidays or when someone that usually works in the office is working from somewhere else.
+
+OOO time has to be requested and approved by your Line Manager and your Project Manager.
+
+There are different types of OOO and each one of them have different requirements for requesting and for being notified.
+
+## Holidays 
+
+As per contract, we are entitled to 25 holiday days per year. You can book your holidays at your convience, you might just need to sync first with your Squad and the other iOS engineers to see if everybody is OK with it. 
+
+Ideally, you should give at least 2 weeks notice before taking any holiday but there are exceptions for emergency situations. If you want to book more than 5 consecutive days you ideally should give 4 weeks notice.
+
+### Steps to request
+
+1. Ask for approval from your PM - only if approved you can proceed to the next steps.
+2. Communicate it to your line manager and mention your PM approval
+3. Request in Bamboo
+
+### Steps to notify
+
+1. [Send an event](#steps-to-configure-the-outlook-event) to the iOS Team Calendar and to your Squad email group
+1. Update the [Team Plan document](https://docs.google.com/spreadsheets/d/1kdY3edy_TeqIGH_7VnZzElxgVo_qD2z4EF-arWNShyw/edit?usp=sharing)
+1. Notify the iOS Team and your Squad via Slack on the working day before you go on holidays
+
+If there is a big impact in the iOS Team or in your Squad due to you being away, please communicate a few days before in the team specific slack channel and do a handover if needed.
+
+#### Steps to Configure the Outlook event
+1. Open Outlook 
+1. Select Calendar
+1. Select Appointment
+1. Select Invite
```

Comment:
"Appointment button then Invite button" is the same as directly hitting the "Meeting" button, so it would be easier to collapse the two steps _(though TIL that one could easily convert an Appointment into a Meeting with that Invite button, very useful since double-click on a slot in the calendar opens the window to create an Appointment, not a Meeting, by default)_